### ISSUE TITLE
Extra `t2` while generating query

### ISSUE THREAD

**Issue by keredson** [keredson/peewee#1]
peewee generating invalid SQL (such as in https://github.com/coleifer/peewee/issues/983) was fixed recently in this minor fork, reported previously in https://github.com/coleifer/peewee/issues/964 and potentially patched in https://github.com/coleifer/peewee/pull/969.  i'm documenting this here in addition to the main site because @coleifer silently deletes comments he disagrees with:
![image](https://cloud.githubusercontent.com/assets/2049665/16057166/65931ddc-323e-11e6-89da-7f5e255e50a2.png)

TL;DR version of why this is an issue:
https://github.com/keredson/peewee#herman-raises-exceptions-when-invalid-columns-are-referenced

@blueset fyi

**Comment by keredson** [comment]
btw this appears to be a long-standing point of confusion w/ peewee.  a quick search reveals:
- https://github.com/coleifer/peewee/issues/987
- https://github.com/coleifer/peewee/issues/913
- https://github.com/coleifer/peewee/issues/787
- https://github.com/coleifer/peewee/issues/665
- https://github.com/coleifer/peewee/issues/446

**Comment by blueset** [comment]
> TL;DR version of why this is an issue:
> https://github.com/keredson/peewee#herman-raises-exceptions-when-invalid-columns-are-referenced

Nice solution and fork!

**Comment by coleifer** [comment]
@keredson -- I'd appreciate it if you would please not spam my ticket tracker with links to the tickets in your fork.

I've made it very clear that I do not want you to be involved in the development of `peewee`. Please respect that decision. I'm happy for you to have forked the code and wish you well.

**Comment by coleifer** [comment]
Here's what I am talking about:

![s1465965554 31](https://cloud.githubusercontent.com/assets/119974/16068259/462ea334-328a-11e6-979b-37de7bcfbd37.png)

**Comment by keredson** [comment]
it's not spam.  it's a collection of issues related by the same root cause.

if you disagree, i recommend using github's "report user" link.  spam is clearly against GH's TOS and they will ban users who do so.

alternatively, if this is just some personal grudge you have against me, i recommend using the "block user" feature, and you'll never have to see me on GH again.

both of these can found on my profile page:  https://github.com/keredson
![image](https://cloud.githubusercontent.com/assets/2049665/16078870/c62a67b2-32c6-11e6-9593-50009254cf56.png)
